geode-redis/src/acceptanceTest/java/org/apache/geode/redis/internal/executor/key/RenameNativeRedisAcceptanceTest.java

```diff
@@ -14,30 +14,25 @@
  */
 package org.apache.geode.redis.internal.executor.key;
 
-import java.util.Random;
-
-import org.junit.BeforeClass;
 import org.junit.ClassRule;
 import org.junit.Ignore;
 import org.junit.Test;
-import redis.clients.jedis.Jedis;
 
 import org.apache.geode.NativeRedisTestRule;
 
-public class RenameNativeRedisAcceptanceTest extends RenameIntegrationTest {
+public class RenameNativeRedisAcceptanceTest extends AbstractRenameIntegrationTest {
+
   @ClassRule
   public static NativeRedisTestRule redis = new NativeRedisTestRule();
 
-  @BeforeClass
-  public static void setUp() {
-    rand = new Random();
-    jedis = new Jedis("localhost", redis.getPort(), 10000000);
-    jedis2 = new Jedis("localhost", redis.getPort(), 10000000);
-    jedis3 = new Jedis("localhost", redis.getPort(), 10000000);
+  @Override
+  public int getPort() {
+    return redis.getPort();
   }
 
   @Override
   @Test
   @Ignore("native redis does implement renamenx")
   public void renamenxIsUnimplemented() {}
+
```

Comment:
Should we move this test out of the `AbstractRenameIntegrationTest` file and into `RenameIntegrationTest` file like the `leakedSubscriptions` test (which is just in the `SubscriptionsIntegrationTest` file)?  Not sure which is better.

Reply:
I decided to just delete the test. I created a story for us to implement `RENAMENX`.

Reply:
sounds good!